two-fer/two_fer.go

```diff
@@ -0,0 +1,20 @@
+// This is a "stub" file.  It's a little start on your solution.
+// It's not a complete solution though; you have to write some code.
+
+// Package twofer should have a package comment that summarizes what it's about.
+// https://golang.org/doc/effective_go.html#commentary
+package twofer
+
+import "fmt"
+
+// Returns the following formatted string depending on the entryString parameter
+// "One for <entryString>, one for me."
+// parameters:
+// * entryString : string to use in the formatting, replaced by "you" if a
+// zero-length string is passed
+func ShareWith(entryString string) string {
+	if entryString == "" {
```

Comment:
in our code base it's more common to write `if len(entryString) == 0`, but both are considered idiomatique (according to [this][1]).

[1]: https://stackoverflow.com/a/18595217/979978

Reply:
Thanks for the review ! This is valuable information, I'll add a regex to my vimrc in order to detect future violations of this codebase practice. Do not hesitate if you have more of them ^^ !